### PR TITLE
Test toArray() more thoroughly

### PR DESCRIPTION
**Pull request by compwright**

### CODE REVIEW

**Comment by compwright** [comment]
Here you go @harikt

File: tests/ManagerTest.php [hunk 1]
```diff
@@ -130,4 +130,34 @@ public function testClear()
...
         }
     }
+
+    public function testToArray()
+    {
+        $relation_builder = new RelationBuilder;
+        $type_builder = new TypeBuilder;
+        $this->manager = new Manager($type_builder, $relation_builder);
+        
+        // toArray() cannot handle circular relationshiops
```
Comment (harikt) [inline tests/ManagerTest.php:140]: this is a drawback I guess.

**Comment by harikt** [comment]
That was quick and thank you for the PR. I want to look more closely before I do a merge. Give me some time. @koriym if you get a chance please do look into this.

**Comment by compwright** [comment]
@pmjones @harikt would be nice to get this merged in and released - thanks in advance.

**Pull request merged**